v2/pkg/engine/datasource/graphql_datasource/graphql_subscription_client.go

```diff
@@ -580,9 +590,16 @@ func (c *subscriptionClient) dial(ctx context.Context, options GraphQLSubscripti
 
 	upgradeResponse, err := c.httpClient.Do(req)
 	if err != nil {
+		// GotConn is not called on error.
```

Comment:
What is the significance of this comment?

Reply:
I think it can be removed, left over from my investigations. Removing